### PR TITLE
Distribution upgrade and security patch

### PR DESCRIPTION
**Pull request by vincentni**
*Issue #, if available:*

*Description of changes:*
upgrade distribution to v2.8.1

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by TerryHowe — APPROVED** [review]
/lgtm

File: projects/distribution/distribution/Makefile [hunk 1]
```diff
@@ -12,7 +12,7 @@ VERSION=$(shell git -C $(REPO) describe --match 'v[0-9]*' --dirty='.m' --always)
 REVISION=$(shell git -C $(REPO) rev-parse HEAD)$(shell if ! git -C $(REPO) diff --no-ext-diff --quiet --exit-code; then echo .m; fi)
 PKG=github.com/docker/distribution
 
-EXTRA_GO_LDFLAGS=-X $(PKG)/version.Version=$(VERSION) -X $(PKG)/version.Revision=$(REVISION) -X $(PKG)/version.Package=$(PKG)
+EXTRA_GO_LDFLAGS=-s -w -X $(PKG)/version.Version=$(VERSION) -X $(PKG)/version.Revision=$(REVISION) -X $(PKG)/version.Package=$(PKG)
 EXTRA_GOBUILD_FLAGS=-tags include_oss,include_gcs
```
Comment (jaxesn) [inline projects/distribution/distribution/Makefile:16]: The s and w should already be being set from the default flags in https://github.com/aws/eks-anywhere-build-tooling/blob/main/Common.mk#L321. was there an issue you saw here?

**Review by jaxesn — CHANGES_REQUESTED** [review]
Could you add an updating section to the readme, we've been gradually doing this as we update projects? Check out the cluster api project readme for an example since it also deals with patches.

**Review by jaxesn — APPROVED** [review]

**Pull request merged**